kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/BrokersResourceIntegrationTest.java

```diff
@@ -115,12 +114,16 @@ public void listBrokers_existingCluster_returnsBrokers() throws Exception {
                             baseUrl
                                 + "/v3/clusters/" + clusterId
                                 + "/brokers/" + nodes.get(2).id()
-                                + "/partition-replicas")))));
+                                + "/partition-replicas"))));
 
     Response response =
         request("/v3/clusters/" + clusterId + "/brokers").accept(Versions.JSON_API).get();
     assertEquals(Status.OK.getStatusCode(), response.getStatus());
-    assertEquals(expected, response.readEntity(String.class));
+
+    ListBrokersResponse actual = OBJECT_MAPPER.readValue(
+            response.readEntity(String.class),
+            ListBrokersResponse.class);
```

Comment:
Here and everywhere else: 

I think this can be simplified as `ListBrokerResponse actual = response.readEntity(ListBrokerResponse.class);`